### ISSUE TITLE
Misspelled 'this.' property accesses don't trigger spelling correction fixes

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/TypeScript#16744]
```ts
class Thing {
    state: any;
    constructor(state) {
        this.state = state;
    }
    doStuff() {
        this.stat/**/;
    }
}

```

![image](https://user-images.githubusercontent.com/972891/27552548-ab2820ee-5a5c-11e7-8cd4-e4d8c3c1fcf7.png)

**Expected**: A quick fix to correct the spelling to 'state'.

**Comment by sandersn** [comment]
Note that the error message gives the correct suggestion; it's just missing the quick fix.

**Comment by sandersn** [comment]
Fix is up at #16844
